Apps/Playground/CMakeLists.txt

```diff
@@ -184,4 +184,5 @@ set_property(TARGET Playground PROPERTY FOLDER Apps)
 source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR}/../BabylonScripts PREFIX Scripts FILES ${BABYLONSCRIPTS})
 source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} FILES ${SCRIPTS})
 source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} FILES ${SOURCES})
+set_property(TARGET Playground PROPERTY VS_DEBUGGER_WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Why do you need this?

Reply:
Don't, forgot to take it out. Should be gone now.